chrome-launcher/chrome-launcher.ts

```diff
@@ -75,19 +96,21 @@ export class ChromeLauncher {
     return flags;
   }
 
-  prepare() {
-    switch (process.platform) {
+  private prepare() {
+    const platform = process.platform as SupportedPlatforms;
+
+    switch (platform) {
       case 'darwin':
       case 'linux':
-        this.TMP_PROFILE_DIR = unixTmpDir();
```

Comment:
lets just move this switch statement into the utils file? and expose a single `makeTmpDir()` over to here.